description.md

```diff
@@ -1,10 +1,14 @@
 
 Sizes can be written as:
 
-- Bytes (Eg. '<span style="color:blue">50</span>' or '<span style="color:blue">800 B</span>')
-- Kilobytes (Eg. '<span style="color:blue">100KB</span>' or '<span style="color:blue">23 K</span>')
-- Megabytes (Eg. '<span style="color:blue">9M</span>' or '<span style="color:blue">34 MB</span>')
-- Gigabytes (Eg. '<span style="color:blue">4GB</span>')
-- Terabytes (Eg. '<span style="color:blue">1 TB</span>')
-- Petabytes (Eg. '<span style="color:blue">0.5PB</span>')
+- Bytes (Eg. '<span style="color:DodgerBlue">50</span>' or '<span style="color:DodgerBlue">800 B</span>')
```

Comment:
Changing from blue to DodgerBlue just lightens it up a little which makes it easier to read.
![image](https://github.com/Josh5/unmanic.plugin.ignore_under_size/assets/4466272/c80eeb62-fa7b-4435-a1d4-f0fa501c7e6d)